### PR TITLE
chore: Update naming to USDS convention (SC-622)

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: src/MainnetController.sol [hunk 1]
```diff
@@ -216,25 +216,25 @@ contract MainnetController is AccessControl {
     /*** Relayer PSM functions                                                                  ***/
     /**********************************************************************************************/
 
-    function swapNSTToUSDC(uint256 usdcAmount) external onlyRole(RELAYER) isActive {
-        uint256 nstAmount = usdcAmount * psm.to18ConversionFactor();
+    function swapUSDSToUSDC(uint256 usdcAmount) external onlyRole(RELAYER) isActive {
+        uint256 usdsAmount = usdcAmount * psm.to18ConversionFactor();
 
-        // Approve NST to DaiNst migrator from the proxy (assumes the proxy has enough NST)
+        // Approve USDS to DaiNst migrator from the proxy (assumes the proxy has enough USDS)
```
Comment (hexonaut) [inline src/MainnetController.sol:222]: `DaiNst` -> `DaiUsds`

File: src/MainnetController.sol [hunk 2]
```diff
@@ -262,13 +262,13 @@ contract MainnetController is AccessControl {
         // Approve DAI to DaiNst migrator from the proxy (assumes the proxy has enough DAI)
```
Comment (hexonaut) [inline src/MainnetController.sol:262]: `DaiNst` -> `DaiUsds`

File: test/mainnet-fork/ForkTestBase.t.sol [hunk 3]
```diff
@@ -165,15 +165,15 @@ contract ForkTestBase is DssTest {
             owner    : PAUSE_PROXY,
             roles    : sharedInst.roles,
             ilk      : ilk,
-            nstJoin  : nstInst.nstJoin
+            usdsJoin : usdsInst.usdsJoin
         });
 
         /*** Step 2: Configure NST, sNST and allocation system ***/
```
Comment (hexonaut) [inline test/mainnet-fork/ForkTestBase.t.sol:171]: Missed rename.

File: test/mainnet-fork/ForkTestBase.t.sol [hunk 4]
```diff
@@ -146,16 +146,16 @@ contract ForkTestBase is DssTest {
 
         /*** Step 1: Deploy NST, sNST and allocation system ***/
```
Comment (hexonaut) [inline test/mainnet-fork/ForkTestBase.t.sol:147]: Missed rename.

File: test/mainnet-fork/PsmCalls.t.sol [hunk 5]
```diff
@@ -5,35 +5,35 @@ import "test/mainnet-fork/ForkTestBase.t.sol";
...
 
         vm.prank(relayer);
         vm.expectRevert("MainnetController/not-active");
-        mainnetController.swapUSDCToNST(1e6);
+        mainnetController.swapUSDCToUSDS(1e6);
     }
 
 }
 
 contract MainnetControllerSwapNSTToUSDCTests is ForkTestBase {
```
Comment (hexonaut) [inline test/mainnet-fork/PsmCalls.t.sol:28]: Missed rename in this file (x4)

File: test/mainnet-fork/VaultCalls.t.sol [hunk 6]
```diff
@@ -3,108 +3,108 @@ pragma solidity ^0.8.21;
...
 
-        assertEq(nst.balanceOf(address(almProxy)), 1e18);
-        assertEq(nst.totalSupply(),                1e18);
+        assertEq(usds.balanceOf(address(almProxy)), 1e18);
+        assertEq(usds.totalSupply(),                1e18);
     }
 
 }
 
 contract MainnetControllerBurnNSTTests is ForkTestBase {
```
Comment (hexonaut) [inline test/mainnet-fork/VaultCalls.t.sol:57]: Missed rename.

File: test/unit/controllers/Constructor.t.sol [hunk 7]
```diff
@@ -6,41 +6,41 @@ import "test/unit/UnitTestBase.t.sol";
...
+        assertEq(address(mainnetController.proxy()),   makeAddr("almProxy"));
+        assertEq(address(mainnetController.vault()),   makeAddr("vault"));
+        assertEq(address(mainnetController.buffer()),  makeAddr("buffer"));
+        assertEq(address(mainnetController.psm()),     address(psm));
+        assertEq(address(mainnetController.daiUsds()), address(daiUsds));
+        assertEq(address(mainnetController.cctp()),    makeAddr("cctp"));
+        assertEq(address(mainnetController.susds()),   address(susds));
+        assertEq(address(mainnetController.dai()),     makeAddr("dai"));   // Dai param in MockDaiUsds
+        assertEq(address(mainnetController.usdc()),    makeAddr("usdc"));  // Gem param in MockPSM
+        assertEq(address(mainnetController.usds()),    makeAddr("usds"));  // Nst param in MockSUsds
```
Comment (hexonaut) [inline test/unit/controllers/Constructor.t.sol:43]: Missed rename.

**Review by hexonaut — CHANGES_REQUESTED** [review]

File: src/MainnetController.sol (hunk 1 above)
Comment (lucas-manuel) [reply src/MainnetController.sol:222]: Updated

File: src/MainnetController.sol (hunk 2 above)
Comment (lucas-manuel) [reply src/MainnetController.sol:262]: Updated

File: test/mainnet-fork/ForkTestBase.t.sol (hunk 4 above)
Comment (lucas-manuel) [reply test/mainnet-fork/ForkTestBase.t.sol:147]: Updated

File: test/mainnet-fork/ForkTestBase.t.sol (hunk 3 above)
Comment (lucas-manuel) [reply test/mainnet-fork/ForkTestBase.t.sol:171]: Updated

File: test/mainnet-fork/PsmCalls.t.sol (hunk 5 above)
Comment (lucas-manuel) [reply test/mainnet-fork/PsmCalls.t.sol:28]: Updated for all

File: test/mainnet-fork/VaultCalls.t.sol (hunk 6 above)
Comment (lucas-manuel) [reply test/mainnet-fork/VaultCalls.t.sol:57]: Updated

File: test/unit/controllers/Constructor.t.sol (hunk 7 above)
Comment (lucas-manuel) [reply test/unit/controllers/Constructor.t.sol:43]: Updated

**Review by hexonaut — APPROVED** [review]

**Pull request merged**